packages/ui/tailwind.config.cjs

```diff
@@ -4,7 +4,6 @@ const { fontFamily } = require('tailwindcss/defaultTheme');
 module.exports = {
   darkMode: ['class'],
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
-  important: true,
```

Comment:
`important: true` caused the accordion animation to break. There does not seem to be a negative effect so far.